launch/gazebo2rviz.launch

```diff
@@ -1,4 +1,10 @@
 <launch>
```

Comment:
Please remove this file from this pull request. I will merge it with https://github.com/andreasBihlmaier/gazebo2rviz/pull/2 where it is also contained.